### PR TITLE
hwdb: fix MXC6655 accelerometer mount matrix for Aquarius Cmp NS483

### PR DESCRIPTION
**Pull request by kovalev0**

### CODE REVIEW

File: hwdb.d/60-sensor.hwdb [hunk 1]
```diff
@@ -118,6 +118,11 @@ sensor:modalias:acpi:BOSC0200*:dmi:*svnAcer*:*pnSpinSP111-34:*
 
 # Aquarius Cmp NS483
 sensor:modalias:acpi:MXC4005*:dmi:*:svnAquarius*:pnCmp*NS483:*
+ ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
```
Comment (bluca) [inline hwdb.d/60-sensor.hwdb:121]: there's no need to duplicate this one, it's identical to the one below, so keep those merged and just add the new one

Comment (kovalev0) [reply hwdb.d/60-sensor.hwdb:121]: okay, then I'll remove the comments about the model so as not to cause confusion

Comment (kovalev0) [reply hwdb.d/60-sensor.hwdb:121]: https://github.com/systemd/systemd/pull/33798

Comment (bluca) [reply hwdb.d/60-sensor.hwdb:121]: just force push to the same branch next time, no need for a different PR

**Pull request closed without merging**